internal/errors/errors.go

```diff
@@ -26,6 +26,9 @@ var Wrapf = errors.Wrapf
 // returns nil.
 var WithMessage = errors.WithMessage
 
+// Error sentinel for invalid source data
+var InvalidSourceData = New("Failed to read all source data during backup.")
```

Comment:
Since this is only used in cmd, wouldn't it be better to just put it cmd_backup.go?

If this comes across as nitpicking: the more stuff goes in internal/restic, the harder it becomes to refactor code. I've hit import cycles on several attempts already.